Business Finder/AppDelegate.h

```diff
@@ -8,10 +8,11 @@
 
 #import <UIKit/UIKit.h>
 #import <CoreData/CoreData.h>
+#import "Model.h"
 
 @interface AppDelegate : UIResponder <UIApplicationDelegate>
 
 @property (strong, nonatomic) UIWindow *window;
-
+@property (strong, nonatomic) Model *model;
```

Comment:
This is a model object that holds state in the AppDelegate, correct? If so, that is a little frown upon within puritanical types. Ways around it? Save that state to disk and retrieve independently throughout the app or pass the state around.